internal/githubrepo/githubrepo.go

```diff
@@ -175,10 +178,20 @@ func (c *Client) GetPullRequestCheckRuns(ctx context.Context, pullRequest *githu
 // GetPullRequestReviews fetches all reviews for the pull request identified
 // by prMetadata.
 func (c *Client) GetPullRequestReviews(ctx context.Context, prMetadata *PullRequestMetadata) ([]*github.PullRequestReview, error) {
-	// TODO(https://github.com/googleapis/librarian/issues/540): implement pagination or use go-github-paginate
-	listOptions := &github.ListOptions{PerPage: 100}
-	reviews, _, err := c.PullRequests.ListReviews(ctx, prMetadata.Repo.Owner, prMetadata.Repo.Name, prMetadata.Number, listOptions)
-	return reviews, err
+	var allReviews []*github.PullRequestReview
+	listOptions := &github.ListOptions{PerPage: pageSize}
```

Comment:
I would pass in pageSize as an argument to GetPullRequestReviews, since this is the API for the client, and set a const in the package where it is called.